lib/cpu_choice.rb

```diff
@@ -3,6 +3,8 @@ class CPUChoice
 
   def initialize(game_over)
     @game_over = game_over
+    @ai = 'O'
```

Comment:
Nice improvement.

Reply:
Github needs to add a plus 1 emoji! But yes nice improvement since you're not using hard-coded strings in your methods. If you needed to change `0` to another symbol you'll now only need to do it in one place.

Reply:
Thanks 🙏🏽